test/lib/twap/util/is_target_met.js

```diff
@@ -6,8 +6,8 @@ const isTargetMet = require('../../../../lib/twap/util/is_target_met')
 
 describe('twap:util:is_target_met', () => {
   it('verifies that the price target is met', () => {
-    assert.ok(isTargetMet({ priceTarget: 1, priceDelta: 0 }, 1))
-    assert.ok(isTargetMet({ priceTarget: 1, priceDelta: 0.2 }, 1.2))
-    assert.ok(!isTargetMet({ priceTarget: 1, priceDelta: 0.2 }, 1.3))
+    assert.ok(isTargetMet({ priceTarget: 5.6, priceDelta: 0 }, 5.6))
```

Comment:
in the best case an additional test case is added, so its clear that it also handles floats and people do not change the test in the future